include/swift/AST/DiagnosticsSema.def

```diff
@@ -551,6 +551,10 @@ ERROR(throws_functiontype_mismatch,none,
       "invalid conversion from throwing function of type %0 to "
       "non-throwing function type %1", (Type, Type))
 
+ERROR(async_functiontype_mismatch,none,
+      "invalid conversion from 'async' function of type %0 to "
+      "synchronous function type %1", (Type, Type))
```

Comment:
"of type" maybe?

Reply:
I sorts like it this way? We trying to convert to a synchronous function type